src/GovUk.Education.ExploreEducationStatistics.Content.Model/Permalink.cs

```diff
@@ -48,5 +49,11 @@ public class Permalink : ICreatedTimestamp<DateTime>
     /// </summary>
     public bool? LegacyHasConfigurationHeaders { get; set; }
 
+    /// <summary>
+    /// Table in universal table format
+    /// </summary>
+    [NotMapped]
+    public dynamic? Table { get; set; }
```

Comment:
> I assume this needs to be dynamic because this is basically html turned into json, which won't neatly fit into an object?

It's in json and it would be possible to define a model for it here. In fact @luke-h1 already did that in an earlier commit but I stripped it out.

The reason I removed it is because it adds an unnecessary overhead to the back-end here. It now doesn't care about the type of table but receives the universal table response from the front-end service, stores it, and responds with it when a permalink is requested.

If there's a breaking change made to the universal format for whatever reason then that's a change to the contract between the front-end services responsible for the generating the table and the front-end component responsible for rendering it.

If we didn't do that and added the type model here as well we'd need to add in support for multiple versions in the model, or run with the latest model and either add in a transformer (like we've currently got for permalinks), or migrate all the data to have the same format (like we've done previously).

This wouldn't be as bad as the situation we had before where the back-end permalink model shared types also used by other back-end services like querying tables, but I feel like it puts us back in that direction which we are trying to move away from with this solution.